src/common/components/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -44,21 +61,38 @@ export function useBreadcrumbs() {
   return useOutletContext<ContextType>();
 }
 
-const Breadcrumbs: React.FC<React.PropsWithChildren<{ breadcrumbs: BreadcrumbListItem[] }>> = ({
+const Breadcrumbs: React.FC<React.PropsWithChildren<{ breadcrumbs: Breadcrumb[] }>> = ({
   breadcrumbs,
 }) => {
   const { t, i18n } = useTranslation();
   const translatedBreadcrumbs = [BREADCRUMBS.homePage, ...breadcrumbs].map((breadcrumb, i) => {
     return {
-      title: t(breadcrumb.title),
+      title: breadcrumb.skipTranslate ? breadcrumb.title : t(breadcrumb.title),
       path:
         breadcrumb.path && i !== breadcrumbs.length
-          ? `/${i18n.language}${t(breadcrumb.path)}`
+          ? breadcrumb.skipTranslate
+            ? breadcrumb.path
+            : `/${i18n.language}${t(breadcrumb.path)}`
           : null,
     };
   });
 
-  return <Breadcrumb ariaLabel={t('common:components:breadcrumb')} list={translatedBreadcrumbs} />;
+  return (
+    <Box backgroundColor="var(--color-white)">
+      <Container alignWithHeader>
+        <Breadcrumb
+          ariaLabel={t('workInstructions:breadcrumb')}
```

Comment:
The translation key could be moved from elsewhere than workInstructions in the json

Reply:
Good point. Also noticed that there was no translation at that path so fixed that.